.gitignore

```diff
@@ -663,3 +663,6 @@ repos/**
 **/xml/**
 repos/
 /resources/_gen/
+
+# Desktop Services Store holds meta information about macOS folders after using Finder
+.DS_Store
```

Comment:
Not enough - as you already ```git add``` for this file ...
So now you need to revert the git addition of this file - along with the .gitignore ;)

Reply:
Added to git-ignore first and then, when I saw result, I committed removal of file.

Reply:
It is a strange little file that auto-generates based on searches in Finder. A real pain...